tetris.js

```diff
@@ -229,49 +242,63 @@ module.exports = {
 		return newArr;
 	},
 	
-	buildTetromino: function(stringOfCoordinates){
-		let math = stringOfCoordinates.split("\n")
-		let arrayOfCoordinates = []
+	getCoordinates: function(stringOfCoordinates){
+		let fieldArray
 		let quantityElem = 0;
+		let arrayOfCoordinates = []
+		if(typeof(stringOfCoordinates) == "string"){
+			fieldArray = stringOfCoordinates.split("\n")
+		}else{
```

Comment:
is it possible for stringOfCoordinates to not be of type "string"?
When?

Reply:
corrected

Reply:
No it's not - otherwise this comment would be marked as "Outdated". If you disagree - attach screenshot